### ISSUE TITLE
Feature Request: Go back in history more than one page at a time with back button

### ISSUE THREAD

**Issue by xhaavardx** [minbrowser/min#552]
I am really missing the ability to look back through my recent history on a certain tab, and then go back x amount of pages. In Chrome, and most browsers for that matter, you can click+hold or right-click the back button to view the recent history, then select one that you want to go back to. 

Like this:
![History-back button](https://lh5.googleusercontent.com/DmqWsbEYGYkXqybrwSsTpxg7q2trH0_hA0j5-Dd2OcUstBDGEXa82PJR_Q-AnM8duSRCxsNn0fYm88pHURPBRhT_xVvQ_fBBAyptrJc5KxjEQo5f0X5rXbIvX-ullFTa32_2OZIq)

**Comment by PalmerAL** [comment]
Makes sense, and that shouldn't be too hard to implement. It would be nice to have a keyboard shortcut for this as well, although I'm not sure what that would be.

**Comment by code-hunger** [comment]
Currently `ctrl + arrow` takes you back & forth. It's intuitive to have `ctrl + alt + arrow` for this new behaviour, at least for me.

**Comment by PalmerAL** [comment]
@code-hunger The shortcut sounds fine; it doesn't seem to conflict with anything else on Mac, but we'll want to check windows and linux as well.

I would agree that new features should be modules. We've kind of been moving in that direction already with the existing components by grouping things together into objects (so ```getWebview``` becomes ```webviews.get```, etc.). Once that's done, it's not too difficult to start importing modules instead of concatenating everything together (in fact, we could probably start doing that for a lot of the components now, it just hasn't been a priority). I don't think refactoring will be quite as difficult as it seems; we have a lot of global variables, but most of them aren't actually used in more than a few places.

**Comment by xhaavardx** [comment]
I don't have any programming input, but I have to say I am genuinely
astonished by the rapid progress and response to this issue. I really
wasn't expecting this. You guys are awesome; keep up the good work! :)

**Comment by PalmerAL** [comment]
@xhaavardx Thank you!
